src/components/Modal/BaseModal.js

```diff
@@ -165,11 +165,7 @@ class BaseModal extends PureComponent {
 
                         return (
                             <View
-                                style={{
-                                    ...styles.defaultModalContainer,
-                                    ...modalContainerStyle,
-                                    ...modalPaddingStyles,
-                                }}
+                                style={[styles.defaultModalContainer, modalContainerStyle, modalPaddingStyles, !this.props.isVisible && styles.pointerEventsNone]}
```

Comment:
If `!this.props.isVisible` is false, then the array will be set to `[styles.defaultModalContainer, modalContainerStyle, modalPaddingStyles, false]`. Do we want that? I'm guessing we don't